frontend/src/reducers/sprints.js

```diff
@@ -32,6 +33,20 @@ export default function sprints(state=initialState, action) {
             localStorage.setItem("boards", JSON.stringify(state.boards));
             return {...state, boardLoading: false};
 
+        case 'PERMISSION_GRANTED':
+            const buttons = {};
+            buttons[action.action] = true;
+            return {...state, buttons: buttons};
+
+        case 'PERMISSION_DENIED':
+            const button_err = {};
+            let data = action.data;
+            if (data.detail) {
+                data = data.detail;
+            }
+            button_err[action.action] = data;
+            return {...state, buttons: button_err};
```

Comment:
I don't think this should be stored as a per-button thing in the global state. 
Instead just set the state value "canCloseSprint" and the button can read that. The global state should not need to be aware of how it's used, or the UI strucutre. In the future this might be tied to other things than a button.